src/WebJobs.Script/Rpc/ILanguageWorkerChannelManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public interface ILanguageWorkerChannelManager
+    {
+        Task InitializeChannelAsync(string language);
+
+        ILanguageWorkerChannel GetChannel(string language);
+
+        Task ReloadEnvironmentVariablesAsync(string language);
+
+        bool ShutdownChannelIfExists(string language);
+
+        Task ShutdownStandbyChannelsAsync();
+
+        Task ShutdownChannelsAsync();
+
+        ILanguageWorkerChannel CreateLanguageWorkerChannel(string workerId, string scriptRootPath, string language, IObservable<FunctionRegistrationContext> functionRegistrations, IMetricsLogger metricsLogger, bool isWebHostChannel, int attemptCount);
```

Comment:
The WebHost naming in this layer isn't correct. WebHost isn't a concept in this WebJobs.Script layer. Can you please rename this throughout?

I'd help come up with a better name, but I still don't understand the concept you're trying to capture with this. But we definitely don't want "WebHost"

Reply:
Makes sense. Code was using this to figure out if function registrations were part of language worker channel creation or not. Updated.